### DISCUSSION TITLE
How can I set up a rule with exact content:

### DISCUSSION THREAD

**Discussion by Londoneye02** [firefly-iii/firefly-iii#8125] [Firefly III]
For example, I want to set up a rule with applies when description contains "BAR", but not Bartolomew...., literally BAR

**Comment by JC5** [comment]
Use the "Description is.." trigger.

**Comment by Londoneye02** [comment]
I could, but i need matches with "BAR ONE" "BAR TWO" "BAR THREE" (so many bars in Spain....) but i don't want match with cabarceno, wich is completely different stuff

**Comment by JC5** [comment]
Add "description is .." and check the NOT box. Unless there are a billion exceptions it should work for you.
